src/main/java/com/rackspace/ceres/app/config/AppProperties.java

```diff
@@ -33,49 +33,49 @@
 @Data
 @Validated
 public class AppProperties {
+
   @NotNull
   @DurationUnit(ChronoUnit.SECONDS)
   Duration rawTtl = Duration.ofHours(6);
 
   /**
-   * When initially creating the Cassandra data table schemas,
-   * this value will be used for the expired data garbage collection.
+   * When initially creating the Cassandra data table schemas, this value will be used for the
+   * expired data garbage collection.
    */
   @Min(60)
   int dataTableGcGraceSeconds = 86400;
 
   /**
    * The approximate number of compaction windows to declare on a newly created raw data table.
-   * Cassandra documentation recommends 20-30 windows:
-   * https://cassandra.apache.org/doc/latest/operating/compaction/twcs.html#twcs
+   * Cassandra documentation recommends 20-30 windows: https://cassandra.apache.org/doc/latest/operating/compaction/twcs.html#twcs
    */
   @Min(20)
   int rawCompactionWindows = 30;
 
   /**
-   * The width of time slots used for partitioning data, which can reduce the number
-   * of Cassandra files that need to be scanned. It is best to set this to the same
-   * as <code>ceres.downsample.timeSlotWidth</code> or greater to ensure downsampling
-   * queries only need to consult one partition.
+   * The width of time slots used for partitioning data, which can reduce the number of Cassandra
```

Comment:
These edits seem unrelated to the topic of this PR? Shouldn't they be in a separate PR?

Reply:
Fixed it.